lib/blocs/group_cubit.dart

```diff
@@ -29,25 +29,3 @@ class GroupCubit extends Cubit<GroupState> implements BaseGroupCubit {
     }
   }
 }
```

Comment:
`GroupCubit` can now be merged with the `BaseGroupCubit`

Reply:
Nice, it has been done!